common/djangoapps/student/views/dashboard.py

```diff
@@ -744,6 +744,14 @@ def student_dashboard(request):
             enr for enr in course_enrollments if entitlement.enrollment_course_run.course_id != enr.course_id
         ]
 
+    # Collect all program types the user is enrolled in
+    enrolled_program_types = []
```

Comment:
For consistency, should we be using the _is_user_enrolled_in_program_type_ method?

Reply:
A quick search looks like we don't actually use this new "enrolled_program_types" context anywhere.

Reply:
Annnnd its gone!